handbook/product/user_research/user_research_observer.md

```diff
@@ -0,0 +1,39 @@
+# So you’re about to help us with user testing
+
+Thank you so much for helping us with our user research! Observing and notetaking during user interviews and testing sessions is an amazing way for you to get a first-hand look at how others use our products and ideas, and your role as a notetaker is a huge help for the team.
```

Comment:
I would add:

Research sessions are not only for designers and product managers. All Sourcegraphers are welcomed to participate! No matter your role, there are valuable learnings you can gain from observing the sessions.